src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1118,6 +1118,30 @@ class LinearIPRDescription(CommonIPR):
     # fmt: on
     # [[[end]]] (checksum: 433953e29d06e22612c935acdbd70db4)
 
+@attr.s(frozen=True, slots=True)
+class VogelIPRDescription(CommonIPR):
+    """
+    .. include:: /alfacase_definitions/VogelIPRDescription.txt
+
+    .. include:: /alfacase_definitions/list_of_unit_for_pressure.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_standard_volume_per_time.txt
+    """
+
+    min_pressure_difference = attrib_scalar(default=Scalar(0.0, "Pa"))
+
+    # [[[cog
+    # cog_out_multi_input("well_max_flow_rate", "standard volume per time", 0.0, "sm3/d")
+    # ]]]
+    # fmt: off
+    well_max_flow_rate_input_type = attrib_enum(default=constants.MultiInputType.Constant)
+    well_max_flow_rate = attrib_scalar(
+        default=Scalar('standard volume per time', 0.0, 'sm3/d')
+    )
+    well_max_flow_rate_curve = attrib_curve(
```

Comment:
Does Vogel's model consider a table of Max Flow Rate?
 I am asking because when we change the Max flow rate we are changing Vogel's model behavior. I mean, we are changing the entire IPR curve of the model during the simulation. Does it make sense? Did you find a reference that presented this kind of approach?

Reply:
I'm not sure about this point. For linear IPR we have a constant value provided by the user to define the slope of the curve. It could be a constant value or transient, leading to a situation where your productivity changes over time, which makes sense as the reservoir pressure will decrease over time, right?

When I was studying Vogel's IPR model, I assumed that q_max could play this same role, which means that it could also vary in time. But I didn't find any reference mentioning it explicitly. Maybe we can allow using just one q_max for the entire simulation to be on the safe side.

Reply:
Humm, you are right. We have it to the Linear IPR model. And also there, the curve(linear) changes over time.

It may make sense in a practical way. Let's ask Carol about it, and see what she has in mind.